src/watcher/watcher.ts

```diff
@@ -121,6 +121,15 @@ export class Watcher extends Component<WatcherAnchorState, IBlockStub & Logs> {
         appointmentState.state === WatcherAppointmentState.OBSERVED &&
         state.blockNumber - appointmentState.blockObserved + 1 >= this.confirmationsBeforeRemoval;
 
+    private shouldRemoveExpiredAppointment = (
```

Comment:
Can you change the name of `shouldRemoveAppointment` to `shouldRemoveObservedAppointment` to avoid ambiguity.

Reply:
Done